Documentation/compatibility/winforms-accessibility-changes-471.md

```diff
@@ -0,0 +1,112 @@
+- UIA Expand/Collapse pattern to `T:System.Windows.Forms.ComboBox` 
+ 
+__Improvements to PropertyBrowser control__
+- Added UIA Expand/Collapse pattern, state and name change notifications and ControlType property with value "Button" to the "Details" button in the error dialog that is displayed when the user enters incorect value in the `T:System.Windows.Forms.PropertyGrid` control 
+- Changed [AccessibleRole](https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.accessiblerole?view=netframework-4.7) of rows in PropertyGrid control from "Row" to "Cell", which maps to UIA ControlType "DataItem", in order to support appropriate keyboard shortcuts and Narrator announcements
+- Improved keyboard navigation between the grid and the ToolBar above it. Pressing "Shift-Tab" now selects the first ToolBar button, instead of the whole ToolBar
+
+__Ensure that OS-defined colors are used in High Contrast themes__
+
+_Note:_ 
+Windows10 had changed values for some high contrast system colors in the *Fall Creator's Update* release. Windows Forms Framework is based on Win32 framework, for he best experience, please run on the latest version of windows and opt in into the latest OS changes by adding app.manifest file in test application and uncommented following code:
+```
+    <!-- Windows 10 -->
+    <supportedOS Id="{8e0f7a12-bfb3-4fe8-b9a5-48fd50a15a9a}" />
+```
+
+- `T:System.Windows.Forms.Button` and `T:System.Windows.Forms.CheckBox` controls with `P:System.Windows.Forms.Control.FlatStyle` set to `F:System.Windows.Forms.FlatStyle.Standard`, which is the default style, are now using OS-defined colors in High Contrast theme in selected state. Before this change test and background colors were not contrasting and were hard to read
+- `T:System.Windows.Forms.Button`, `T:System.Windows.Forms.CheckBox`, `T:System.Windows.Forms.RadioButton`, `T:System.Windows.Forms.Label`, `T:System.Windows.Forms.LinkLabel` and `T:System.Windows.Forms.GroupBox` with`P:System.Windows.Forms.Control.Enabled` set to _false_, used a shaded color to rendered text in High Contrast themes, resulting in low contrast against the background. Now these controls use "Disabled Text" color defined by the OS. This fix applies to control with `P:System.Windows.Forms.Control.FlatStyle` set value other than `F:System.Windows.Forms.FlatStyle.System`, the later controls are rendered by the OS.
+
+__Improved keyboard navigation__
+
+- When `T:System.Windows.Forms.ComboBox` control has `P:System.Windows.Forms.ComboBox.DropDownStyle` set to `F:System.Windows.Forms.DropDownStyle.DropDownList`, and is the first control in the TabOrder on the form, it is now displaying focus rectangle when the parent form is opened using the keyboard. Before this change, keyboard focus was on this control, but focus indicator was not rendered.
```

Comment:
When --> When a 
has --> has its 
set to --> property set to
and is --> and it is
it is now displaying --> it now displays a 
Before this change, --> Previously, the
focus indicator --> the focus indicator